admin_guide/production_sizing.rst

```diff
@@ -0,0 +1,38 @@
+.. _production_sizing:
+
+Production Sizing
+$$$$$$$$$$$$$$$$$
+
+This section explores configuration settings for running a cluster in production.
+
+.. _prod_shard_count:
+
+Shard Count
+===========
+
+The number of nodes in a cluster is easy to change (see :ref:`scaling_out_cluster`), but the number of shards to distribute among those nodes is more difficult to change after cluster creation. Choosing the shard count is a balance between the flexibility of having more shards, and the overhead for query planning and execution across them.
```

Comment:
Could we clarify that number of shards is for an existing distributed table? (not as a whole cluster-wide)?